### DISCUSSION TITLE
Closure of the Classic INSPIRE Geoportal harvesting console

### DISCUSSION THREAD

**Discussion by jrc-inspire** [INSPIRE-MIF/helpdesk-geoportal#152] [Announcements]
Dear Community, 

As reported in the last 75 MIG-T meeting, the rollout of the revamped INSPIRE Geoportal based on GeoNetwork is approaching.

As part of this rollout the service of the [Classic Geoportal harvesting console](https://inspire-geoportal.ec.europa.eu/harvestconsole) - and its related tools - will be definitively closed on Thursday 20 July - From that date onwards all harvesting processes shall be performed using the [revamped GeoNetwork console](https://jrc-geoportal-internal.geocat.live/geonetwork).

**Comment by vicehovskis** [comment]
Hello,

How to get access to the revamped GeoNetwork console? I can`t log in with old Harvest console credentials.

**Comment by jescriu** [comment]
Dear @vicehovskis,
The new credentials for accessing the new GeoNetwork harvesting console were submitted to LV representatives / responsibles for the national discovery service on 12 December 2022, in view of using this new system already for the past Monitoring and Reporting round.
We will forward you this information again.
Thank you.

**Comment by fdumortier** [comment]
Hello, the revamped GeoNetwork console https://jrc-geoportal-internal.geocat.live/geonetwork is unreachable. What's wrong? Kind regards, François Du Mortier

**Comment by jescriu** [comment]
Dear @fdumortier,
Nothing is wrong. 
Please note this announcement published in the INSPIRE Geoportal helpdesk this morning:
https://github.com/INSPIRE-MIF/helpdesk-geoportal/discussions

Thank you for your patience. It is in the interest of all of us to achieve the revamped Geoportal rollout.

**Comment by jescriu** [comment]
Please, if not done yet start watching this repository.
Once done you will be in the watchers list and should be receiving notifications, which will make you easier to follow the repository:
https://github.com/INSPIRE-MIF/helpdesk-geoportal/watchers
